packages/ckeditor5-engine/src/view/downcastwriter.js

```diff
@@ -1626,10 +1626,18 @@ export default class DowncastWriter {
 	 * 	@returns {Boolean} Returns `true` if elements are merged.
 	 */
 	_wrapAttributeElement( wrapper, toWrap ) {
-		if ( !canBeJoined( wrapper, toWrap ) ) {
```

Comment:
Do we still need this change?

We do not rely on wrapping attr elements anymore.

Reply:
Nope, we don't need it.